src/dal/Jalasoft.TeamUp.Resumes.DAL/Jalasoft.TeamUp.Resumes.DAL.csproj

```diff
@@ -21,10 +21,4 @@
 	  <ProjectReference Include="..\Jalasoft.TeamUp.Resumes.DAL.Interfaces\Jalasoft.TeamUp.Resumes.DAL.Interfaces.csproj" />
```

Comment:
I comment here since I can't do it above. In lines 10 and 11 please double-check this are necessary otherwise remove them

Reply:
Remove them.